WolvenKit.App/ViewModels/Documents/RedDocumentViewToolbarModel.cs

```diff
@@ -460,7 +460,14 @@ public string GetTextureDirForDependencies(bool useTextureSubfolder)
             {
                 // if the folder contains more than one .mesh file, add a subdirectory inside "textures"
                 var fileName = Path.GetFileName(CurrentTab.FilePath.Split('.').FirstOrDefault() ?? "").ToFileName();
-                destFolder = Path.Combine(dirName, "textures", fileName);
+                if (!dirName.Contains("textures"))
```

Comment:
This got smuggled in from the other branch now

Reply:
not smuggled - I decided to take it over, because it's now no longer arbitrary or hidden :)